source/app/controllers/teams_controller.rb

```diff
@@ -103,6 +100,10 @@ def manage_member
 
   private
 
+  def role_based_dashboard_path
+    path send("teams_#{current_user.role}_dashboard_index_path", course_id: @selected_course.id)
```

Comment:
The path method is not defined and will cause a NoMethodError. It should be removed: 

```suggestion
    send("teams_#{current_user.role}_dashboard_index_path", course_id: @selected_course.id)
```